_includes/homepage/announcements.html

```diff
@@ -0,0 +1,53 @@
+<section class="bp-section {{ gray_container_class }}">
+  <div class="bp-container is-fluid">
+    <div class="row">
+      <div class="col is-full p-16">
+        {%- if section.announcements.subtitle -%}
+        <p class="eyebrow is-uppercase pb-4 has-text-centered">
+          {{- section.announcements.subtitle -}}
+        </p>
+        {%- endif -%} {%- if section.announcements.title -%}
+        <h1 class="has-text-secondary pb-4 has-text-centered">
+          <b>{{- section.announcements.title -}}</b>
+        </h1>
+        {%- endif -%}
+
+        <hr class="my-8 announcements-divider" />
+
+        {%- for announcement in section.announcements.announcement_items -%}
+        <div class="row is-desktop announcements-announcement-row-px-desktop">
+          <div
+            class="col is-4-desktop announcements-announcement-col-px-desktop announcements-announcement-title-mr-desktop"
+          >
+            <h3 class="announcements-announcement-title mb-4">
+              <b>{{- announcement.title -}}</b>
+            </h3>
+            <p class="announcements-announcement-subtitle">
+              {{- announcement.date -}}
+            </p>
+          </div>
+          <div class="col announcements-announcement-col-px-desktop">
+            <p>{{- announcement.announcement -}}</p>
```

Comment:
not a blocker, but just note that those stuff without conditionals need to be guaranteed by our CMS